src/cockpit/channels/filesystem.py

```diff
@@ -154,6 +154,7 @@ class FsReplaceChannel(Channel):
     payload = 'fsreplace1'
 
     _path = None
+    _send_acks = None
```

Comment:
Oops.  That should drop.  Surprised that vulture didn't find and complain about it.  Maybe the `_` :(